modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/configuration/RedirectConfiguration.java

```diff
@@ -53,6 +53,6 @@ public interface RedirectConfiguration {
 		description = "check-redirect-not-found-entries-interval-help",
 		name = "check-redirect-not-found-entries-interval", required = false
 	)
-	public int checkRedirectNotFoundEntriesInterval();
+	public float checkRedirectNotFoundEntriesInterval();
```

Comment:
Hi @holatuwol 
We think that the use of a float to let the user introduce not integer numbers is not a good idea and it's no other use on the system like this.
The configuration help says:
´Set the time interval, in hours, between 404 URL checks. All 404 URLs stored in the database that exceed both the configured maximum age and maximum number will be removed.`

We do not understand the need of the check of the redirections for their removal on periods of time so shorts. But if this is needed we think that id better to use int and minutes as another configurations.

Reply:
Understood, I'll go ahead and close the LPS as won't fix.